.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -18,21 +18,86 @@ AGENTS_LABELS = [
     "acc-ubuntu-18.04-vanilla": env.UBUNTU_VANILLA_1804_CUSTOM_LABEL ?: "vanilla-ubuntu-1804"
 ]
 
+def CmakeArgs(String build_type = "RelWithDebInfo", String code_coverage = "OFF", String debug_malloc = "ON", String lvi_args="", String cmake_args = "") {
+    def args = "-G Ninja -DCMAKE_INSTALL_PREFIX:PATH='/opt/openenclave' -DCPACK_GENERATOR=DEB -DCODE_COVERAGE=${code_coverage} -DUSE_DEBUG_MALLOC=${debug_malloc} -DCMAKE_BUILD_TYPE=${build_type} ${lvi_args} ${cmake_args} -Wdev"
+    return args
+}
+
+def TestCommand() {
+    def buildAndTest = """
+        echo "Running Test Command"
+        ctest --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
+    """
+    return buildAndTest
+}
+
+def InstallBuildCommand() {
+    def localPackage = """
+        echo "Running Install Build Command"
+        cpack -G DEB
+        sudo killall -9 apt && sudo killall -9 dpkg
```

Comment:
When we manually test, killing the apt/dpkg processes are usually fine. But for automated tests we want to make sure we know that the state of our environment is stable. Killing apt/dpkg processes may introduce flakiness by leaving the system in a state that has different package versions as the apt process is killed at different points in time across different builds, and therefore different from builds with the exact same parameter.

I don't see any apt lock issues with our existing Linux builds, but if you encounter this and wait to proceed past this error you should look into waiting for the lock to be released.  For example, I implemented a function for Ansible installations [here in #3869](https://github.com/openenclave/openenclave/pull/3869/files).